controllers/phish.go

```diff
@@ -126,6 +127,32 @@ func (ps *PhishingServer) registerRoutes() {
 	ps.server.Handler = phishHandler
 }
 
+// ArbitraryEventHandler deals with arbitrary events - for example opening Word documents, secondary links, etc
```

Comment:
Can we call them "Custom Events", e.g. `CustomEventHandler`?

Reply:
Yeah that does sound better! I've done a `s/Arbitrary/Custom/g`

Also I can spell 'Custom' more easily than I can spell 'Arbitrary'